Sources/MapboxNavigation/CarPlayMapViewController.swift

```diff
@@ -358,7 +358,6 @@ open class CarPlayMapViewController: UIViewController {
         view.setNeedsUpdateConstraints()
         
         guard let activeRoute = navigationMapView.routes?.first else {
```

Comment:
Navigation camera should not switch back to the following mode when safe area insets change (e.g. when navigation bar becomes hidden).

Reply:
Because if the camera is idle and there're routes on the map, it may be in the routes preview mode. Because by default, all the routes is displayed through `navigationMapView.showcase(_:routesPresentationStyle:animated:)` in `CarPlayManager` with all fit option, should we also keep the all fit option in the `CarPlayMapViewController`?

Reply:
Yep, you're right, not sure why I've modified this part (switching to the following mode will have to be removed though).